### PR TITLE
Code Quality: Renamed SideBar with Sidebar

### PR DESCRIPTION
**Pull request by 0x5bfa**
## Summary

The first step to introduce the SidebarView to props window & settings dialog.

- Format codebase
- SideBar -> Sidebar

## PR Checklist

- [x] Were these changes approved in an issue or discussion with the project maintainers? In order to prevent extra work, feature requests and changes to the codebase must be approved before the pull request will be reviewed. This prevents extra work for the contributors and maintainers.
   Closes #issue...
- [x] Did you build the app and test your changes?
- [ ] Did you check for accessibility? You can use Accessibility Insights for this.
- [ ] Did you remove any strings from the en-us resource file?
   - [ ] Did you search the solution to see if the string is still being used? 
- [ ] Did you implement any design changes to an existing feature?
   - [ ] Was this change approved?
- [ ] Are there any other steps that were used to validate these changes?
   _None_

## Screenshots (optional)

_None_

### CODE REVIEW

File: src/Files.App/UserControls/Sidebar/SidebarStyles.xaml [hunk 1]
```diff
@@ -158,15 +162,7 @@
...
 
-							<FontIcon
-								x:Name="ExpanderIcon"
-								Grid.Column="2"
-								Margin="8,0,14,0"
-								AutomationProperties.AccessibilityView="Raw"
-								FontSize="8"
-								Glyph="&#xE70E;"
-								Visibility="Collapsed" />
-
```
Comment (yaira2) [inline src/Files.App/UserControls/Sidebar/SidebarStyles.xaml:169]: Was this removed by mistake?

File: src/Files.App/UserControls/Sidebar/SideBarView.xaml [hunk 2]
```diff
@@ -59,10 +61,11 @@
 				<CompositeTransform x:Name="PaneColumnGridTransform" />
 			</Grid.RenderTransform>
 
+			<!--  Main Items  -->
 			<ItemsRepeaterScrollHost HorizontalAlignment="Stretch" VerticalAlignment="Stretch">
 				<ScrollViewer
 					x:Name="MenuItemHostScrollViewer"
-					Padding="2,8,0,2"
+					Padding="0,8,0,0"
```
Comment (yaira2) [inline src/Files.App/UserControls/Sidebar/SideBarView.xaml]: The padding is specific, please revert this change.

File: src/Files.App/UserControls/Sidebar/SidebarStyles.xaml (hunk 1 above)
Comment (0x5bfa) [reply src/Files.App/UserControls/Sidebar/SidebarStyles.xaml:169]: This is not used. I've ensured in removing.
I think this was created in a dev stage.

File: src/Files.App/UserControls/Sidebar/SideBarView.xaml (hunk 2 above)
Comment (0x5bfa) [reply src/Files.App/UserControls/Sidebar/SideBarView.xaml]: But how come?

This is annoying.
image

Comment (yaira2) [reply src/Files.App/UserControls/Sidebar/SideBarView.xaml]: I don't recall right now but I think it had to do with not overlapping the resize strip.

Comment (0x5bfa) [reply src/Files.App/UserControls/Sidebar/SideBarView.xaml]: @marcelwgn Do you remember? This file [hasn't been updated](https://github.com/files-community/Files/blame/main/src/Files.App/UserControls/SideBar/SideBarView.xaml#L65) yet ever since first SidebarView implemented.

Comment (0x5bfa) [reply src/Files.App/UserControls/Sidebar/SideBarView.xaml]: At this time, I'm gonna revert because this is not in the scope.

Comment (0x5bfa) [reply src/Files.App/UserControls/Sidebar/SideBarView.xaml]: @yaira2 your call then.

Comment (0x5bfa) [reply src/Files.App/UserControls/Sidebar/SideBarView.xaml]: @mdtauk fyi

Comment (yaira2) [reply src/Files.App/UserControls/Sidebar/SideBarView.xaml]: As a sidenote, design changes are out of scope for this PR.
